packages/visx-scale/test/scaleRadial.test.ts

```diff
@@ -17,7 +17,8 @@ describe('sclaeRadial()', () => {
 
   it('set unknown', () => {
     const scale = scaleRadial({ domain: [0, 10], unknown: 'green' });
-    expect(scale('sandwich' as any)).toEqual('green');
+    // coerce to make TS happy
+    expect(scale('sandwich' as unknown as number)).toEqual('green');
```

Comment:
wait, what?

`scale` seems to only expect a number. you're passing in `sandwich` and forcing typescript to be ok with it. Shouldn't this test be deleted since TS makes sure this can never happen?

Or, if you want to test the unknown values, can we pass in a number that's not in the domain to test that? either `-1` or `11`?